packages/ui/src/Radio.module.scss

```diff
@@ -0,0 +1,109 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.wrapper {
+  display: inline-flex;
+  // inline-flex changes vertical align compared to display: flex
+  // setting vertical-align allows us to easily use radio button
+  // inline e.g. next to the text
+  // Most common use case will be still the usage in a form where parent
+  // wrapper controls this.
+  vertical-align: bottom;
+  align-items: center;
+  cursor: pointer;
+
+  @include c.typographyBodySmall;
+  // we need to provide spacing between radios if they're in a vertical column
+  margin-bottom: c.$grid;
```

Comment:
Isn't this spacing going to add the bottom margin to the last radio? Do we always want to do it? Should we consider something in a fashion of https://github.com/hazelcast/frontend-shared/pull/54/files#r519910049?

Reply:
I actually prefer to have it there for consistency, I like to think of it as part of the element and when I placed some text under a radio group it felt natural to me. Don't you think we could leave it as is?

Reply:
@jvorcak I get the sane defaults argument. I am just a little uncomfortable with the lest radio always having the bottom margin.
@danad02 @jankoritak @jozefhruska WDYT?
Personally, I feel like adding top-level margins affects the external layout - the layout of the parent container and it, in my view, kind of, breaks component encapsulation. My mental model is that each component should take care of its visuals and its internal layout only, leaving the external layout to the parent. However, I am open to being convinced otherwise. After all, we all will be using this lib. We all should be comfortable with it.

Reply:
This is how it looks with and without it.
![Screenshot 2020-11-10 at 16 12 44](https://user-images.githubusercontent.com/1083817/98693240-54be5080-2370-11eb-819f-67ccd5aebb3f.png)
![Screenshot 2020-11-10 at 16 12 48](https://user-images.githubusercontent.com/1083817/98693245-55ef7d80-2370-11eb-844a-1cc1e5be157c.png)

Reply:
I think it would be better for parent (`RadioGroup` in this case) to handle this margin.

Reply:
https://github.com/hazelcast/frontend-shared/pull/54/commits/5ac6232bfc914d761f7501e0551ed77dde24a7bc

Reply:
@jvorcak this small proposal could help us restore the spacing for the error https://github.com/hazelcast/frontend-shared/pull/54/files#r521216637